pkg/karmadactl/cmdinit/karmada/webhook_configuration.go

```diff
@@ -256,6 +256,23 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: [ "v1" ]
     timeoutSeconds: 3
+  - name: resourcedeletionprotection.karmada.io
```

Comment:
```suggestion
  - name: resource deletion protection
```

Reply:
we may use the consist name like the other webohooks?
or `resource-deletion-protection`
<img width="788" alt="image" src="https://github.com/karmada-io/karmada/assets/19872346/867976dc-714e-4db5-9901-72006c68cc49">
<img width="1296" alt="image" src="https://github.com/karmada-io/karmada/assets/19872346/1b06edf2-748b-4e17-8ae1-45aaa03952ef">

Reply:
Yeah, you are right, I didn't noticed that the name should be fully qualified.
Then, please ignore my suggestions. Let's keep the `resourcedeletionprotection.karmada.io`.